google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -115,13 +123,21 @@ appengine.stop.modules.version.success.dialog.title=Application Stopped
 appengine.stop.modules.version.canceled.message=Undeploy Canceled
 appengine.deployment.error.during.staging=Deployment failed due to staging failing with exit code: {0}.
 appengine.deployment.exception.during.staging=Deployment failed due to an exception while staging the project.
+appengine.deployment.error.staging.yaml=The specified YAML configuration file does not exist or is not a valid file.
+appengine.deployment.error.staging.dockerfile=The specified Dockerfile configuration file does not exist or is not a valid file.
+appengine.deployment.error.staging.gotosettings=Set a valid file in Module Settings or use another one.
 appengine.deployment.error.not.logged.in=You must be logged in to perform this action.
+appengine.deployment.error.suggestflexfacet=If you can't find your artifacts in the deployment run configuration, consider adding Google App Engine Flexible support to your module.
 appengine.deployment.source.not.found.error=Deployment source not found: {0}.
 appengine.deployment.invalid.source.error=Invalid deployment source.
 appengine.deployment.status.deploying=Deploying to App Engine
 appengine.deployment.error.invalid.cloudsdk=Invalid Cloud SDK directory path.
+appengine.deployment.error.deployable.notjarorwar=The selected archive must have either JAR or WAR format.
```

Comment:
The file chooser link doesn't let them, but they're able to manually specify a file location, too.